### ISSUE TITLE
Notifications.scheduleNotificationAsync API trigger passing Date in iOS is not working

### ISSUE THREAD

**Issue by cHaLkdusT** [expo/expo#7937]
## 🐛 Bug Report
Using bare React Native project using expo-notifications dependency. iOS doesn't work when using Notifications.scheduleNotificationAsync API and uses Date object for the trigger parameter, but is working in Android

### Environment

> Expo CLI 3.13.8 environment info:
>     System:
>       OS: macOS 10.15.4
>       Shell: 5.7.1 - /bin/zsh
>     Binaries:
>       Node: 12.13.0 - /usr/local/bin/node
>       Yarn: 1.22.0 - /usr/local/bin/yarn
>       npm: 6.14.4 - /usr/local/bin/npm
>       Watchman: 4.9.0 - /usr/local/bin/watchman
>     IDEs:
>       Android Studio: 3.5 AI-191.8026.42.35.5977832
>       Xcode: 11.3.1/11C504 - /usr/bin/xcodebuild
>     npmPackages:
>       react: 16.11.0 => 16.11.0
>       react-native: 0.62.1 => 0.62.1
>     npmGlobalPackages:
>       expo-cli: 3.13.8

App target: Android and iOS

### Steps to Reproduce
```
    // Local notification after 5 seconds
    const trigger = new Date(Date.now() + 5 * 1000);

    Notifications.scheduleNotificationAsync({
      content: {
        title: 'After 5 seconds',
      },
      trigger,
    });
```

### Expected Behavior
Local notification should prompt in both Android and iOS

### Actual Behavior
Local notification works only in Android but not in iOS

### Reproducible Demo
N/A

### Other
As a workaround on my current project, I used this code snippet
```
const trigger = new Date(Date.now() + 5 * 1000);
Notifications.scheduleNotificationAsync({
     content: {
       title: 'Trigger with Date object',
     },
     trigger: Platform.OS === 'android' ? date : date.getTime() / 1000,
   });
```

**Comment by sjchmiela** [comment]
Published `expo-notifications@0.1.2` with the fix. Try upgrading to the latest version! Sorry for the bug!

**Comment by cHaLkdusT** [comment]
Thanks mate! 🍻
